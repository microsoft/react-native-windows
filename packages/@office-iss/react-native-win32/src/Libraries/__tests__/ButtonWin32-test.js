
'use strict';

const React = require('react');
const ReactTestRenderer = require('react-test-renderer');

const {ButtonWin32} = require('../Components/Button/ButtonWin32');

describe('ButtonWin32', () => {
  it('renders native component', () => {
    const component = ReactTestRenderer.create(
      <ButtonWin32/>
    );
    expect(component).toMatchSnapshot();
  });
});