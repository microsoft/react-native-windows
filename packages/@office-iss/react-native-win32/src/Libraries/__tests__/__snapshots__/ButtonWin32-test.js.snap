// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonWin32 renders native component 1`] = `
<View
  accessibilityRole="button"
  accessibilityState={
    Object {
      "disabled": undefined,
      "selected": false,
    }
  }
  accessible={true}
  focusable={true}
  onBlur={[Function]}
  onFocus={[Function]}
  onTouchEnd={[Function]}
>
  <Text
    textStyle="None"
  />
</View>
`;
